index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio: Setup and Mobile First</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
+    <nav>
+      <a href="#" class="logo">My Logo</a>
+      <svg viewBox="0 0 100 80" width="40" height="40" class="svg-menu">
+        <rect y="38" width="100" height="8" rx="8" fill="#6070FF"></rect>
+        <rect y="58" width="100" height="8" rx="8" fill="#6070FF"></rect>
+        <rect y="78" width="100" height="8" rx="8" fill="#6070FF"></rect>
+      </svg>
+    </nav>
+  
+    <div class="intro">
+      <h1>I'm Raphael.<br /> I am so glad to see you!</h1>
+    </div>
+  
+    <div class="main">
+      <p>I'm a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+  
+    <div class="connect text-left">
+      <p class="social-media">LET'S CONNECT</p>
+      <div class="social-icons">
+        <a href="#"><img src="./images/icons/twitter.svg" alt="Twitter"></a>
+        <a href="#"><img src="./images/icons/Linkedin.png" alt="LinkedIn"></a>
+        <a href="#"><img src="./images/icons/m.svg" alt="Medium"></a>
+        <a href="#"><img src="./images/icons/github.svg" alt="GitHub"></a>
+        <a href="#"><img src="./images/icons/random.svg" alt="Random"></a>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ]  The container of this section suppose to be a `section` not `div` tag. It represents a generic standalone section of the document. 👍